core/spec/models/spree/tax/item_adjuster_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+RSpec.describe Spree::Tax::ItemAdjuster do
+  subject(:adjuster) { described_class.new(item) }
+  let(:order) { Spree::Order.new }
+  let(:item) { Spree::LineItem.new(order: order) }
+
+  before do
+    allow(order).to receive(:tax_zone) { build(:zone) }
+  end
+
+  describe 'initialization' do
+
+    it 'sets @order to adjustable.order' do
+      expect(adjuster.instance_variable_get(:@order)).to eq(item.order)
+    end
+
+    it 'sets adjustable' do
+      expect(adjuster.instance_variable_get(:@item)).to eq(item)
+    end
+  end
```

Comment:
I'd rather stick to testing only public interface of a class. The below `#adjust` should fail of those are not being correctly set. Otherwise, you introduce coupling between the code and the tests : )